### PR TITLE
Feature/main page

### PR DESCRIPTION
**Pull request by Wperricone**
## What does this PR actually DO? / Issue you solved
- This PR adds a rudimentary landing page as well as a Nav bar component. Also implemented styled components. 

## Type of Changes:
- [x] New feature
- [ ] Bug fix
- [ ] Refactor

## Where are the Changes:
- We added a landing page and started the nav bar. We also took out some of the Cypress example tests.

## How was this tested:
- Looking in the Browser

## What are the relevant tickets:
-

### CODE REVIEW

File: src/Components/App.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      </div>
+    </div>
+  );
+}
+
+export default App;
+
+const Img = styled.img`
+width: 100vw;
+`;
```
Comment (stephanie-roe) [inline src/Components/App.js:23]: Nice implementation of styled components! I look forward to seeing how this makes our codebase more streamlined as our application expands in scope.

File: src/Components/NavBar.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+					<h3>All Crafts</h3>
+				</Button>
+				<Button>
+					<h3>Profile</h3>
+				</Button>
+			</div>
+		</Nav>
+	)
+}
+
```
Comment (stephanie-roe) [inline src/Components/NavBar.js:22]: I feel like indentation here is a little extreme. Super small detail, but we might be able to clean this up a bit.

File: src/Components/NavBar.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styled from 'styled-components';
+
+
+
+const NavBar = () => {
+	return (
+		<Nav>
+			<img>
+			</img>
```
Comment (Abumpus1) [inline src/Components/NavBar.js:10]: Once we have a logo we can add that to the nav bar here. Will need to make sure to make it a self closing tag.

**Review by stephanie-roe — APPROVED** [review]
This is looking great so far! I'm glad we were able to do some GraphQL and CircleCI setup without too much challenge. This week, let's focus on expanding this skeleton and building out all of our key components and functionalities.

**Pull request merged**